packages/alfa-rules/test/sia-r62/rule.spec.tsx

```diff
@@ -41,45 +40,79 @@ test(`evaluate() passes an <a> element with a <p> parent element with non-link
   ]);
 });
 
-test(`evaluate() fails an applicable <a> element that removes the default text
-      decoration on hover and focus and is determined not to be distinguishable`, async (t) => {
+test(`evaluate() fails an <a> element that removes the default text decoration
+      without replacing it with another distinguishing feature`, async (t) => {
   const target = <a href="#">Link</a>;
 
   const document = Document.of(
     [<p>Hello {target}</p>],
     [
       h.sheet([
-        h.rule.style("a:hover, a:focus", {
+        h.rule.style("a", {
```

Comment:
We should keep some cases (likely not this one) where the "bad" styling is only here in one of the 6 states. It is a design choice to require distinguishability in each of these states (rather than only on focus/hover), so having test cases will avoid regressions on this choice.

Reply:
Good point 👍